tools/qontract_cli.py

```diff
@@ -517,6 +517,18 @@ def saas_dev(ctx, app_name=None, saas_file_name=None, env_name=None):
                 parameters.update(saas_file_parameters)
                 parameters.update(rt_parameters)
                 parameters.update(target_parameters)
+
+                for replace_key, replace_value in parameters.items():
+                    if not isinstance(replace_value, str):
+                        continue
+                    replace_pattern = '${' + replace_key + '}'
+                    for k, v in parameters.items():
+                        if not isinstance(v, str):
+                            continue
+                        if replace_pattern in v:
+                            parameters[k] = \
+                                v.replace(replace_pattern, replace_value)
```

Comment:
Not sure I understand what you're trying to achieve here. Mind explaining?

Reply:
if someone uses the following parameter: `AUTHORIZE_URL: ${OCM_BASE_URL}/auth`
we replace the `OCM_BASE_URL` with the actual value coming from the environment.
this is actually the parameter unpacking.

Reply:
I see. `validate saas files are reusing environment parameters` or the commit messges do not tell the full story here :)
tks.